index.html

```diff
@@ -72,6 +72,30 @@ <h1 class="worktext">My Recent Works</h1>
                <li> <a href="#">Get My Resume</a></li>
             </ul>
         </div>
+        <footer>
+            <p class="context">I'm always interested in hearing about new projects, 
+                so if you'd like to chat please get in touch.</p>
+            <form action="https://formspree.io/f/xwkzpzlo" id="contactform" method="post">
+                <ul id="listdetails">
+                   <li id="namefield">
+                    <!-- <label for="name" id="namelabel">Full name</label> -->
+                    <input type="text" id="name" name="fullname" required maxlength="30" placeholder="Full name"/>
+                </li>
+                <li id="emailfield">
+                    <!-- <label for="mail" id="maillabel">Email address</label> -->
+                    <input type="email" id="mail" name="useremail" required placeholder="Email address"/>
+                </li>
+                <li id="messagefield">
+                    <!-- <label for="message">Enter text here</label> -->
+                    <textarea id="message" name="usermessage" required maxlength="500" placeholder="Enter text here"></textarea>
+                </li>
+                <li class="buttonlabel">
+                    <button class="fbutton" type="submit"><ins class="btext">Get in touch</ins></button>
+                </li>
+                </ul>
+            </form>
+            
+        </footer>
```

Comment:
- [ ] Please kindly make sure your design stick as much as possible to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471), also kindly fix the empty space to the right of your website 🙃 🙃 
- [ ] Please kindly add the socials media icons to the end of your website 😉

Your design
<img width="347" alt="Screen Shot 2022-08-30 at 18 08 56" src="https://user-images.githubusercontent.com/74501523/187490498-48e48ad7-9d55-4790-b86d-01904a921179.png">

Figma design
<img width="280" alt="Screen Shot 2022-08-30 at 18 08 09" src="https://user-images.githubusercontent.com/74501523/187490606-32a08806-6598-441b-8460-9af2311d2d73.png">